lib/workers/manual.js

```diff
@@ -31,11 +31,14 @@ module.exports = class ManualWorker {
   }
 
   // eslint-disable-next-line class-methods-use-this
-  async ready () {
-    return Bluebird.resolve()
+  async ready (os) {
+    await os.runFetch()
```

Comment:
We shouldn't be using `.ready()` to initialize the OS. This function is meant to hold initialization logic specific to a worker type (ie something we need to do before actually using the TestBot).

The `.flash()` method already takes an image file, and blindly flashes that, so these `os` function calls should live in the code that orchestrates all these classes, rather than on a particular class.